tests/snapshots/tests.contrib.rq.test_rq.test_enqueue_distributed_tracing_enabled_None_worker_service_custom-worker-service.snap

```diff
@@ -0,0 +1,45 @@
+[[{"name" "rq.queue.enqueue_job"
+   "service" "rq"
+   "resource" "tests.contrib.rq.jobs.job_add1"
+   "type" "worker"
+   "error" 0
+   "span_id" 0
+   "trace_id" 0
+   "parent_id" nil
+   "start" 1631053633351328100
+   "duration" 2363000
+   "meta" {"runtime-id" "b1fd435dff564be79ebf4876f19e3401"
+           "queue.name" "q"
+           "job.id" "24d5cedf-caa3-41a1-9763-32a79f1b2ca1"
+           "job.func_name" "tests.contrib.rq.jobs.job_add1"}
+   "metrics" {"_dd.agent_psr" 1.0
+              "_sampling_priority_v1" 1
+              "system.pid" 2666
+              "_dd.tracer_kr" 1.0}}
+      {"name" "rq.worker.perform_job"
+       "service" "custom-worker-service"
+       "resource" "tests.contrib.rq.jobs.job_add1"
+       "type" "worker"
+       "error" 0
+       "span_id" 1
+       "trace_id" 0
+       "parent_id" 0
+       "start" 1631053636948434900
+       "duration" 63030800
+       "meta" {"runtime-id" "1a0fb63e68d94c12a364d691b87726c5"
```

Comment:
we ignore these when asserting right? might be interesting if we could assert that these came from two different processes (runtime-id and system.pid) since that is what we expect.

aside/musing, not 100% related to this PR